class.two-factor-core.php

```diff
@@ -86,7 +86,11 @@ public static function get_providers() {
 			 * Confirm that it's been successfully included before instantiating.
 			 */
 			if ( class_exists( $class ) ) {
-				$providers[ $class ] = call_user_func( array( $class, 'get_instance' ) );
+				try {
+					$providers[ $class ] = call_user_func( array( $class, 'get_instance' ) );
+				} catch ( Exception $e ) {
+					echo esc_html( sprintf( __( 'Could not get provider instance: %s', 'two-factor' ), $class ) );
```

Comment:
This is called during `init` hook. So, I'm concerned that error messages might be displayed before <!DOCTYPE>